src/DIPS.Xamarin.UI/Controls/Modality/ModalityLayout.xaml.cs

```diff
@@ -165,6 +184,11 @@ private Frame CreateOverlay()
         /// <remarks>Using relative to parent with the constraints will give you the <see cref="ModalityLayout"/></remarks>
         public void Show(IModalityHandler modalityHandler, View view, Constraint? xConstraint = null, Constraint? yConstraint = null, Constraint? widthConstraint = null, Constraint? heightConstraint = null)
         {
+            if (ShouldCloseOpenedModals && CurrentShowingModalityLayout != modalityHandler)
+            {
+                HideCurrentShowingModality();

```

Comment:
This would close all kinds of modals right? FloatingMenu and PopUps. Is this intended?

Reply:
Yes, as discussed with @haavamoa, this behavior would allow only one modal open in a ModalityLayout.